File: Python/tests/test_igt_vpp_sigmainvgam.py
import pytest

from hbayesdm.models import igt_vpp_sigmainvgam


def test_igt_vpp_sigmainvgam():
    _ = igt_vpp_sigmainvgam(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_igt_vpp_sigmainvgam.R
context("Test igt_vpp_sigmainvgam")
library(hBayesDM)

test_that("Test igt_vpp_sigmainvgam", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(igt_vpp_sigmainvgam(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
